### DISCUSSION TITLE
Setup mailserver for 1 domain and not recieving any mails ever since

### DISCUSSION THREAD

**Discussion by Jarrodsz** [docker-mailserver/docker-mailserver#3356] [Q&A]
Setup 1 mailserver for 1 domain.
I have port 587 open

Can send myself mail but nit recieving any other mail and complaints people cannot reach me.

Is there a way to resolve this or debug this?

Im in digital ocean and port 25 is blocked.

Even a lousy hosting peovider i have to pay 10 dollar per mail
Domain can host decent mailbox why  cant i do the same.

**Comment by casperklein** [comment]
Port 25 is essential. Without, you won't be able to receive regular emails.

**Reply by Jarrodsz** [reply]
Can you guys suggest a hosting provider that allows me port 25 then? 

I seem to
Be unable to receieve any mails even while we discussed before port 587 should work most of the time after 3 weeks not a single mail was recieved. Only the mails that where send to myself.

Its really frustrating you cannot send a list of domains and business details to open port 25 up.

Digitalocean dis not respond to my request.

I read linode.com allows for opening port 25 if you specify a list of domains intension and business data. This is not a problem for me to provide these details.

Perhaps its a good idea to have a dedicated documentation page with a list of hosters that allow poet 25 to be opened with there conditions?

I "only" want to own my own mails and run mailboxes on my own companys ( i build online startups ) 

I hope i can resolve it somehow moving away from digitalocean where they allow me to have port 25 open.

This mailserver getting to work has been really cybersome. Really want to get this resolved asap.

If you guys ( and galls) all tun your own mailsever there must be a provider you all use that allow you port 25?

**Reply by Jarrodsz** [reply]
Hetzner.com is my last resort

**Reply by casperklein** [reply]
There are plenty (Contabo, Netcup, OVH, Strato). Shouldn't be too hard to find one 😉
